### DISCUSSION TITLE
Strimzi Kafka Installation step by step

### DISCUSSION THREAD

**Discussion by bharathreddy901** [strimzi/strimzi-kafka-operator#8888] [General]
Hi Team,

I am new to Strimzi Kafka, Is there any document to install Strimzi Kafka on k8s. I have deployed strimzi operator through helm charts. After that I need to deploy Kafka. Can you please share the location Kafka yaml files or helm charts to deploy it.

Thanks

**Comment by scholzj** [comment]
Did you check the docs? https://strimzi.io/documentation/
